aptos-move/framework/aptos-framework/sources/configs/stake.move

```diff
@@ -752,31 +738,19 @@ module aptos_framework::stake {
         pool_address: address,
         amount: u64,
         owner_cap: &OwnerCapability,
-    ) acquires StakePool, StakePoolEvents, ValidatorSet {
+    ) acquires StakePool, StakePoolEvents {
         assert!(owner_cap.pool_address == pool_address, error::invalid_argument(ENOT_OWNER));
 
         // Short-circuit if amount to unlock is 0 so we don't emit events.
         if (amount == 0) {
             return
         };
 
+        // Unlocked coins are moved to pending_inactive. When the current lockup cycle expires, they will be moved into
+        // inactive in the earliest possible epoch transition.
         let stake_pool = borrow_global_mut<StakePool>(pool_address);
         let unlocked_stake = coin::extract<AptosCoin>(&mut stake_pool.active, amount);
-
-        // Withdrawal is not allowed if lockup has not expired.
```

Comment:
This is no longer needed as stake is moved from active to pending_inactive and not inactive so it's not withdrawable. Pending_inactive funds are only moved to inactive in the first epoch after the current lockup expires.